src/libopensc/cards.h

```diff
@@ -231,6 +231,7 @@ enum {
 
 	/* EstEID cards */
 	SC_CARD_TYPE_ESTEID_2018 = 35000,
+	SC_CARD_TYPE_ESTEID_2018_V2,
```

Comment:
we are not very strict with this, but I think you should add it behind the 2025 variant...

Reply:
There will be 2025 card also in spring.
SC_CARD_TYPE_ESTEID_2018_V2_2025 ?

Reply:
This would assure that the number for `SC_CARD_TYPE_ESTEID_2025` doesn't change:
```
	SC_CARD_TYPE_ESTEID_2018 = 35000,
	SC_CARD_TYPE_ESTEID_2025,
	SC_CARD_TYPE_ESTEID_2018_V2,
```